lib/scope.js

```diff
@@ -1052,8 +1030,11 @@ module.exports = {
         }
 
         // Try to set other types of fields
-        let was_set = await scope.setVariable( scope, { field, var_data, from_table });
-        if ( was_set ) {
+        let row_used = await scope.setVariable( scope, { field, var_data: ensured_var_data });
+        if ( row_used ) {
+          // Esp important for `.target_number`
+          if ( row_used.source ) { row_used.source.times_used += 1 }
+          else { row_used.times_used += 1; }
```

Comment:
Won't all rows with `target_number` as a variable have a source attribute? When will the `else` part of this be used?

Reply:
This might be one of two questions, so I'll answer both:

If the question is about when `.times_used` will be used:
1. I plan to use it in reports to list the rows there were used vs. the ones that weren't
2. I already don't love the idea of only some rows having some props, like `source`. I'd like to keep them as consistent as possible.

If it's meant literally, this happens with all rows, so this will be used with all non `target_number` rows as well.

Thoughts?

Reply:
Ah I didn't realize we wanted to track times used for all rows, not just target_number rows. 👍